### PR TITLE
5.1 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,101 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Added the ability to reset user emails in System Console > Users.
+ 
+#### Permissions
+ - Made Permissions Reset CLI command custom-role aware.
+ 
+#### Command Line Interface (CLI)
+ - Have permanent delete user CLI command delete FileInfos for that user's posts.
+ - Improved behaviour when running the CLI command outside of the bin directory.
+ 
+#### xxxx //// category?
```
Comment (cpanato) [inline source/administration/changelog.md]: need fix this section

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,93 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+#### Support Mattermost on a subpath
+ - Added support for hosting Mattermost at any route (e.g., https://www.example.com/mm) with newly added subpath support.
+ 
+#### CSV Compliance Export
+ - Removed existing Compliance Reports feature and replaced it with an improved Compliance Export feature that captures more data in CSV format.
+
+### Improvements
+
+#### Web User Interface
+ - Added member count for the direct message list.
```
Comment (jasonblais) [inline source/administration/changelog.md]: Which list?

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,93 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Changed all instances of "Delete Channel" to "Archive Channel".
+ - Added Purpose as a searchable field.
+ 
+#### Administration
+ - Added the ability to reset user emails in System Console > Users.
+ - Added a prototype for CBA.
+ - Don't require a server restart to run the job server for the first time.
+ 
+#### Command Line Interface (CLI)
+ - Made Permissions Reset CLI command custom-role aware.
```
Comment (jasonblais) [inline source/administration/changelog.md]: Not sure what this means?

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,93 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Have permanent delete user CLI command delete FileInfos for that user's posts.
```
Comment (jasonblais) [inline source/administration/changelog.md]: Not sure what this means?

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,6 +4,93 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Improved behaviour when running the CLI command outside of the bin directory.
```
Comment (jasonblais) [inline source/administration/changelog.md]: What is the improved behaviour?

File: source/administration/changelog.md [hunk 6]
```diff
@@ -4,6 +4,93 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+### Bug Fixes
+
+ - Fixed an issue where iOS could not reply to a push notification.
```
Comment (jasonblais) [inline source/administration/changelog.md]: Not sure what this means

File: source/administration/changelog.md [hunk 7]
```diff
@@ -4,6 +4,93 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Fixed an issue with an incorrect system message after converting a public channel to private.
+ - Fixed an issue with being unable to add emoji reactions after expanding the message details sidebar.
+ - Fixed an issue where Rate limit settings could not be edited in the System Console, and weren't displayed in the User Interface if configured via config.json.
+
+### Compatibility
+
+#### Deprecated Features
```
Comment (jasonblais) [inline source/administration/changelog.md]: None?

File: source/administration/changelog.md (hunk 5 above)
Comment (amyblais) [reply source/administration/changelog.md]: https://mattermost.atlassian.net/browse/MM-6839

File: source/administration/changelog.md (hunk 3 above)
Comment (amyblais) [reply source/administration/changelog.md]: https://mattermost.atlassian.net/browse/MM-10570

File: source/administration/changelog.md (hunk 4 above)
Comment (amyblais) [reply source/administration/changelog.md]: https://mattermost.atlassian.net/browse/MM-10830

File: source/administration/changelog.md (hunk 7 above)
Comment (amyblais) [reply source/administration/changelog.md]: Not sure, where can I find info?

File: source/administration/changelog.md (hunk 2 above)
Comment (amyblais) [reply source/administration/changelog.md]: https://mattermost.atlassian.net/browse/MM-5761

Comment (jasonblais) [reply source/administration/changelog.md]: Looks like that ticket is open and I don't see a count in the direct messages "more" menu, so I'll remove it

File: source/administration/changelog.md (hunk 7 above)
Comment (jasonblais) [reply source/administration/changelog.md]: Let's add a process to ask the PM and dev release manager in release discussion channel to confirm if there are any deprecated features. I think we do that for breaking changes as well.

There aren't any this release though.

Comment (amyblais) [reply source/administration/changelog.md]: Ok

**Comment by cpanato** [comment]
I did not see any updates for database/api/websocket

**Comment by amyblais** [comment]
@cpanato You can ask Derrick how he used to check them or you can ask other devs if they know of any known changes.

**Pull request merged**